apps/src/StudioApp.js

```diff
@@ -578,10 +592,15 @@ StudioApp.prototype.scaleLegacyShare = function() {
   applyTransformScale(vizContainer, 'scale(' + scale + ')');
 };
 
-StudioApp.prototype.substituteInstructionImages = function(htmlText) {
+/**
+ * @param {string} htmlText
+ * @param {Object.<string, string>} [substitutions] Dictionary strings (keys) to
```

Comment:
I was going to question whether `substitutions` is really optional, but I tried it and discovered that `for (var x in undefined)` and `for (var x in null)` are safe.   TIL.